src/index.ts

```diff
@@ -69,20 +86,6 @@ setImmediate(async function setUpAndMonitor(): Promise<void> {
   await sendClusterMetadata();
   await loadAndSendWorkloadEventsPolicy();
   await monitor();
-
-  const interval: number = 4 * 60 * 60 * 1000; // 4 hours in milliseconds
-  if (config.SYSDIG_ENDPOINT && config.SYSDIG_TOKEN) {
-    setInterval(async () => {
-      try {
-        await scrapeData();
-      } catch (error) {
-        logger.error(
-          { error },
-          'an error occurred while scraping runtime data',
-        );
-      }
-    }, interval).unref();
-  } else {
-    logger.info({}, 'Sysdig integration not detected');
-  }
+  await setupSysdigIntegration();
+  await setupHealthCheck();
```

Comment:
Should the healtherCheck be set up inside the `setupSysdigIntegration`?

Reply:
I wanted to make it a bit more generic, so it's not only about Sysdig but also about other factors in the future!